daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-bulk.md

```diff
@@ -269,6 +269,127 @@ Invoke-RestMethod -Method Post -ContentType 'application/json' -Uri 'http://loca
 
 {{< /tabs >}}
 
+## Subscribing messages in bulk
+
+The bulk subscribe API allows you to subscribe multiple messages from a topic in a single request. 
+As we know from [How to: Publish & Subscribe to topics]({{< ref howto-publish-subscribe.md >}}), there are two ways to subscribe to topic(s):
+
+- **Declaratively**, where subscriptions are defined in an external file.
+- **Programmatically**, where subscriptions are defined in user code.
+
+To Bulk Subscribe to topic(s), we just need to use `bulkSubscribe` spec attribute, something like following:
+
+```yaml
+apiVersion: dapr.io/v2alpha1
+kind: Subscription
+metadata:
+  name: order-pub-sub
+spec:
+  topic: orders
+  routes: 
+    default: /checkout
+  pubsubname: order-pub-sub
+  bulkSubscribe:
+    enabled: true
+    maxMessagesCount: 100
+    maxAwaitDurationMs: 40
+scopes:
+- orderprocessing
+- checkout
+```
+
+In the example above, `bulkSubscribe` altogether is _optional_. If you decide to use `bulkSubscribe`, then:
+- `enabled` is mandatory
```

Comment:
describe what this does. Does this apply per topic or for all topics. Something like this. 
```suggestion
- `enabled` is mandatory and enable or disables bulk subscriptions on this topic
```

Reply:
Done